cloud.foundry.cli/build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
     // This dependency is used internally, and not exposed to consumers on their own compile classpath.
     implementation 'com.google.guava:guava:28.0-jre'
 
-    compile 'info.picocli:picocli:4.2.0'
+    compile 'info.picocli:picocli:4.3.0'
```

Comment:
We also need to update the Bill-of-materials and replace the according version number.

Reply:
BOM has been updated :)